core/types/collateral.go

```diff
@@ -28,6 +80,15 @@ type Account struct {
 	Type     AccountType
 }
 
+func (a Account) ToDetails() *AD {
+	return &AD{
```

Comment:
Can we name AD as AccountDetails ?

Reply:
We can. I just liked better AD as it’s shorted and I thought I’d use it more in the codebase / it looks slightly like ID as well, which was it relates too

Reply:
addressed.